### PR TITLE
RFP: ars-js-filecoin-explorer by ARS team

### PR DESCRIPTION
**Pull request by waynewyang**
Name of Project: ars-js-filecoin-explorer

### CODE REVIEW

File: rfp-proposals/ars-js-filecoin-explorer.md [hunk 1]
```diff
@@ -0,0 +1,350 @@
...
+
+In order to meet needs of different users, we divide the filecoin blockchain explorer into four functional dimensions, 
+
+- Blockchain
+  - Including  Tipset, Block, Message, Actor.
+- Token
+  - Transaction history
+  - Token History
+    - Plan block rewards
+    - Real block rewards
```
Comment (pooja) [inline rfp-proposals/ars-js-filecoin-explorer.md]: Why would planned and real block rewards be different? Can you say more about what this means?

File: rfp-proposals/ars-js-filecoin-explorer.md [hunk 2]
```diff
@@ -0,0 +1,350 @@
...
+    - Liquid token 
+    - Collateral token
+    - Account that held token
+- Storage Market
+  - Storage miner
```
Comment (pooja) [inline rfp-proposals/ars-js-filecoin-explorer.md]: What information would you reveal about a storage miner?

File: rfp-proposals/ars-js-filecoin-explorer.md [hunk 3]
```diff
@@ -0,0 +1,350 @@
...
+    - The pie chart
+  - Trend chart of storage ask
+    - abscissa: Time(24hour,Day,Week,Month,Year)
+    - ordinate:
+      - The line chart:  ask price (min,max,avg,start,end)
+      - histogram:  
+        - duration active miners counts
+        - duration dealed capacity
+- 5 RetrievalMarket
+  - Trend chart of retrieval miners ，power&collateral
```
Comment (pooja) [inline rfp-proposals/ars-js-filecoin-explorer.md]: This is an assumption about the retrieval market -- we won't have retrieval miners with power on mainnet launch.

File: rfp-proposals/ars-js-filecoin-explorer.md [hunk 4]
```diff
@@ -0,0 +1,350 @@
...
+# Additional Information
+
+**Now, We've alreay do some pre work**,
+
+- [Demo code of ars-js-filecoin-explorer](https://github.com/arsyun/js-filecoin-explorer)
+  - Front-end: React
+  - Back-end: Nodejs
+- [ Backend db design of ars-js-filecoin-explorer ](https://github.com/arsyun/js-filecoin-explorer/tree/master/db)
+  - Mongodb, Read > Write.
+- [Ue draft of ars-js-filecoin-explorer ]( https://wbhnrp.axshare.com )
```
Comment (pooja) [inline rfp-proposals/ars-js-filecoin-explorer.md]: Great to see that some pre-work has already been done. However, the mockups look like they focus more on network activity than producing a chain explorer.

File: rfp-proposals/ars-js-filecoin-explorer.md (hunk 3 above)
Comment (waynewyang) [reply rfp-proposals/ars-js-filecoin-explorer.md]: I'll update the PR, --we won't have retrieval miners with power on mainnet launch.

File: rfp-proposals/ars-js-filecoin-explorer.md (hunk 4 above)
Comment (waynewyang) [reply rfp-proposals/ars-js-filecoin-explorer.md]: It's just a draft UE, actually our explorer including chian explorer and network activity. At present, the chain explorer  is in Network Overview,we'll  perfect it later.

File: rfp-proposals/ars-js-filecoin-explorer.md (hunk 1 above)
Comment (waynewyang) [reply rfp-proposals/ars-js-filecoin-explorer.md]: There was theoretical release curve for FIL, It perherps can't match with actual release counts every day.

File: rfp-proposals/ars-js-filecoin-explorer.md (hunk 2 above)
Comment (waynewyang) [reply rfp-proposals/ars-js-filecoin-explorer.md]: - You can refer to 
https://github.com/arsyun/js-filecoin-explorer/tree/master/db#StorageMiner
- If you want to know about more infomation ,you can view the db design
https://github.com/arsyun/js-filecoin-explorer/tree/master/db
```
storageMiner{
	_id:ObjectID("xxxxxxxxxxxx"),
	nickName:"arsyun",
	address:"t2u2dppg6lczbsukzufyagqkaofbiic7gaqec7izy",//index
	peerID:"QmeRygishJkb4MDF8FTg4Qxuoy8nJmoUcAxVSSwKNmcY2k",
	creatTime:2019-09-18 09:09:56
	
	active:true,
	ipAddr:113.12.2.11,
	location:ShenZhen,	
	ttl:300ms,
	power:5%,
	capacity:2T,
	blockPercentage:4.9%,
	parentHashes:"bafy2bzaceboap5lhhus773fkswxwflxfg4b2arkilgmdjh74vedvt34kzevto",
	blockHeight:1111,	
	blockTime:"",
	isInConsensus:true,
	lastSeenTime:"",
	reputation:"",
	totalReward:2000FIL,
	totalServiceIncome:3000FIL
}
```

**Comment by pooja** [comment]
We've finished the wave 1 grant selection process. Can you please email devgrants@filecoin.org so we can share our decision and next steps, if any? In your email, please include a link to your proposal PR so we know who we're talking to!

Thanks,
Filecoin DevGrants team

**Comment by pooja** [comment]
Greetings,

Thank you for your application to the Filecoin DevGrants program. We appreciate your proposal!

You were not selected for a grant in Wave 1, but we hope that you consider applying again for a future DevGrant in a later wave.

We selected grants based on:
- what makes sense to build now based on where the Filecoin node implementations are
- the likelihood of your team being able to work independently of the Filecoin engineering teams working on the core client implementations
- degree of refactoring your proposal may require as the software clients evolve
- the integrity of the proposal and near-term value to the Filecoin ecosystem
- your commitment to the ecosystem and desire to maintain your project longer-term

As the Filecoin implementations advance towards Testnet and Mainnet, we hope you continue to consider new proposals for projects. We would love to see more applications from you in the future.

Please stay tuned for more Filecoin grant RFPs. :)

**Pull request closed without merging**